lib/rubocop/cop/rails/create_table_with_timestamps.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks the migration for which timestamps are not included
+      # when creating a new table.
+      # In many cases, timestamps are useful information and should be added.
+      #
+      # @example
+      #   # bad
+      #   create_table :users
+      #
+      #   # bad
+      #   create_table :users do |t|
+      #     t.string :name
+      #     t.string :email
+      #   end
+      #
+      #   # good
+      #   create_table :users do |t|
+      #     t.string :name
+      #     t.string :email
+      #
+      #     t.timestamps
+      #   end
+      class CreateTableWithTimestamps < Cop
+        MSG = 'Add timestamps when creating a new table.'.freeze
+
+        def_node_matcher :create_table_with_block?, <<-PATTERN
+          (block
+            (send nil? :create_table ...)
+            (args (arg _var))
+            begin)
+        PATTERN
+
+        def_node_matcher :create_table_call?, <<-PATTERN
+          (send nil? :create_table ...)
+        PATTERN
+
+        def_node_matcher :timestamps?, <<-PATTERN
+          (send _var :timestamps ...)
+        PATTERN
+
+        def on_send(node)
+          return unless create_table_call?(node)
+          ancestor = node.ancestors.first
+
+          if create_table_with_block?(ancestor)
+            unless ancestor.body.child_nodes.any? { |child| timestamps?(child) }
```

Comment:
We can use `def_node_search` instead of `child_nodes.any?`.
For example: 

https://github.com/bbatsov/rubocop/blob/848d34c4818ce7d1a5b1dbd778b49262af8d1182/lib/rubocop/cop/bundler/duplicated_gem.rb#L48